network/network.html

```diff
@@ -0,0 +1,99 @@
+ <head>
+    <title>Wifi Network</title>
+    <meta charset="utf-8">
+    <link href="../base1/cockpit.css" type="text/css" rel="stylesheet">
+    <script src="../base1/jquery.js"></script>
+    <script src="../base1/cockpit.js"></script>
+</head>
+<body>
+	<button class="network"  id="1" style="margin-left: 20px; margin-top: 20px;"> <font size="2" face="sans-serif"> Change SSID </font></button>
```

Comment:
leave style (here and on the other places) to a .css file instead of inlining it here. It makes much easier to then maintain.  Why are we setting the font like that?